components/teams-service/storage/postgres/datamigration/sql/03_update_admin_team_name.up.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+UPDATE teams
```

Comment:
this change will persist if the user resets to v1, but we decided this was ok since we already have the editor and viewer teams persisting after reset.

Reply:
If they are in v2.x mode, any new datamigrations should happen teams service restart (which happens on upgrade):

https://github.com/chef/automate/blob/master/components/teams-service/storage/postgres/postgres.go#L45-L49

Reply:
@tylercloke We're a little confused on your comment. (Also we've moved this to be a permanent migration anyway so maybe it's moot)